packages/vim.rb

```diff
@@ -3,79 +3,119 @@
-            --enable-perlinterp \
-            --enable-python3interp \
-            --enable-rubyinterp \
-            --with-python3-command=#{CREW_PREFIX}/bin/python3 \
-            --with-ruby-command=#{CREW_PREFIX}/bin/ruby \
-            --with-features=huge \
-            --disable-selinux"
-    system 'make'
+    system "./configure",
+              "--prefix=#{CREW_PREFIX}",
+              "--localstatedir=#{CREW_PREFIX}/var/lib/vim",
+              "--with-features=huge",
+              "--with-compiledby='Chromebrew'",
+              "--with-x=no",
+              "--disable-gui",
+              "--enable-multibyte",
+              "--enable-cscope",
+              "--enable-perlinterp=dynamic",
+              "--enable-pythoninterp=dynamic",
+              "--enable-python3interp=dynamic",
+              "--enable-rubyinterp=dynamic",
+              "--disable-selinux"
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81"
   end
 
   def self.install
-    configdir = "#{CREW_PREFIX}/.config/vim"
-    configdestdir = "#{CREW_DEST_PREFIX}/.config/vim"
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81", "DESTDIR=#{CREW_DEST_DIR}", "install"
+
     system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{configdestdir}"
-    system "cp -r #{CREW_DEST_PREFIX}/share/vim/vim81/* #{configdestdir}"
-    FileUtils.mkdir_p "#{CREW_DEST_HOME}"
-    FileUtils.mkdir_p "#{configdir}" unless Dir.exists? "#{configdir}"
-    system "ln -s #{configdir} #{CREW_DEST_HOME}/.vim"
-    system "ln -s #{configdir}/vimrc #{CREW_DEST_HOME}/.vimrc"
-    system "touch #{configdir}/vimrc #{configdestdir}/vimrc"
+
+    # remove desktop and icon files for the terminal package
+    system "rm", "-r", "#{CREW_DEST_PREFIX}/share/applications"
+    system "rm", "-r", "#{CREW_DEST_PREFIX}/share/icons"
+
+    # add some defaults simulating XDG support
+    FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/share/vim/vimfiles")
+    File.write("#{CREW_DEST_PREFIX}/share/vim/vimfiles/chromebrew.vim", <<~EOF)
+      " Global vimrc - setting some sane defaults
+      "
+      " DO NOT EDIT THIS FILE. IT'S OVERWRITTEN UPON UPGRADES.
+      "
+      " Use #{CREW_PREFIX}/etc/vimrc for system-wide and ~/.vimrc for personal
+      " configuration.
+
+      " Use Vim defaults instead of 100% vi compatibility
+      " Avoid side-effects when nocompatible has already been set.
+      if &compatible
+        set nocompatible
+      endif
+
+      set backspace=indent,eol,start
+      set ruler
+      set suffixes+=.aux,.bbl,.blg,.brf,.cb,.dvi,.idx,.ilg,.ind,.inx,.jpg,.log,.out,.png,.toc
+      set suffixes-=.h
+      set suffixes-=.obj
+
+      " Move temporary files to a secure location to protect against CVE-2017-1000382
+      if exists('$XDG_CACHE_HOME')
+        let &g:directory=$XDG_CACHE_HOME
```

Comment:
This variable will require the xdg_base dependency.  Better make sure this variable is not expanded.  The `$` might need escaped.

Reply:
It will be used if it's set, `xdg_base` is optional in this case as we'll fallback to `~/.cache` and required directories will be created.